src/main/java/ro/developmentfactory/Cinema/OptimisticLocking.java

```diff
@@ -19,7 +19,8 @@ public void bookSeats(int numSeats) throws NotEnoughSeatsException,SeatsAlreadyR
         while(!occupiedSeats.compareAndSet(currentOccupiedSeats, currentOccupiedSeats + numSeats)) {
             currentOccupiedSeats = occupiedSeats.get();
         }
-            }
+        return false;
```

Comment:
usually, we return true when something succeeds and false when failed, so it would be nice to return true

Reply:
`PessimisticLocking.bookSeats()` still is void